src/EditorFeatures/TestUtilities/Threading/StaTaskScheduler.cs

```diff
@@ -18,6 +18,40 @@ public sealed class StaTaskScheduler : IDisposable
 
         public bool IsRunningInScheduler => StaThread.ManagedThreadId == Thread.CurrentThread.ManagedThreadId;
 
+        // An int to allow use of Interlocked.CompareExchange
+        private static int s_SubscribedToAppDomainUnload = 0;
```

Comment:
💡 This field can be removed since single-execute is already a guarantee of the static constructor.

Reply:
D'oh, forgot to delete.